pkg/segment/writer/segstore.go

```diff
@@ -124,6 +128,22 @@ func NewSegStore(orgId uint64) *SegStore {
 	return segstore
 }
 
+func (ss *SegStore) GetNewBitset(bsSize uint) *bitset.BitSet {
+	if ss.bsPoolCurrIdx >= ss.bsPoolCount {
+		newCount := uint32(1000)
```

Comment:
nit: Can we make this value as a constant?

Reply:
done